src/components/AccountSecurity/LoginPage.tsx

```diff
@@ -182,6 +198,7 @@ class LoginPage extends Component<Props, State> {
           this.props.alert.show('Network Failure: Check Server Connection');
           this.setState({ buttonState: '' });
         });
+      this.resetRecaptcha();
```

Comment:
@vanessa-hu can you actually move the resetRecaptcha to inside the fetch call under each clearInput()? 
There are sometimes strange behaviors if you move it outside the fetch call, because of the unknown order of execution